concat-js.php

```diff
@@ -217,7 +217,7 @@ function do_items( $handles = false, $group = false ) {
 						return $path;
 					}, $js_array['paths'] );
 					$mtime = max( array_map( 'filemtime', $paths ) );
-					$path_str = implode( ',', $js_array['paths'] ) . "?m=${mtime}j";
```

Comment:
This is horrible. :laughing:

Reply:
I've seen that happen mistakenly with trying to use vim movement while in insert mode... thankfully, this one already existed in the code we built upon. 😄